classroom_server_api/plumber.R

```diff
@@ -0,0 +1,385 @@
+source(here::here("classroom_server_api/helper_funcs.R"))
+
+# TODO: disallow interacting with archived classroom? (Or do on app side?)
+
+# Do some config/connect to DB
+cfg <- config::get("database", file = "config.yml")
+con <<- do.call(pool::dbPool, cfg)
```

Comment:
You know, I was having trouble at some point...but looks like it works fine with non-global assignment now.

Reply:
Runs fine without global assignment if just hit run, but breaks when I run as background job. Wat?

Reply:
That's kinda insane... has to be something about the bootstrapping of plumber... sad! We can leave this as is for now. I would add a TODO or issue that we could dig into later to understand why. This may be something plumber needs to improve.